### ISSUE TITLE
App crashes on launch and preferences are erased after update to 1.19.0  

### ISSUE THREAD

**Issue by shushtain** [minbrowser/min#1558]
- **Min Version:** 1.19.0
- **Operating system:** Windows 10

## Expected Behavior
Update and successful run :)

## Actual Behavior
Started to receive error alerts after installing the update. Also all my settings were gone including "ads blocked" counter, but not history and logins. At first 2 launches it created 5 processes of Min, 42% RAM were used, so those are not just sub-processes, I assume. Then the app crashed.

![Screenshot (165)](https://user-images.githubusercontent.com/58133782/114264029-388f1000-99f1-11eb-9732-c8e884916b88.png)

Then it launched without an error. I customized the browser again, and set shortcuts with Cyrillic letters as well (!), and it doesn't crash, but gives me alerts on launch again, although they are not as persistent (only once per launch).
(And just crashed again after opening settings to take a screenshot...)

## To Reproduce
I don't know why it crashes for me on launch. I guess the real question is what are those lines in your code for. Hope you will be able to fix it with ease. Have a good day :)

**Comment by shushtain** [comment]
No user scripts are enabled, no password manager, no proxy, no bangs.

**Comment by shushtain** [comment]
It crashes when I open a blank tab, and close it. If I go to some page, it doesn't crash on closing the tab.

**Comment by shushtain** [comment]
Now it crashes on every tab being closed directly (not from the task manager). I don't know if this depends on the amount of tabs. Maybe it's because now I have the amount that makes them shrink. Idk what that error really means. Just try to make some sense of this bug.

**Comment by PalmerAL** [comment]
That's a very strange list of issues; it sort of looks like your settings file is corrupted. Can you try this?

1. Go to `\Users\<your username>\AppData\Roaming\Min`, and find the `settings.json` file.
2. Open it and paste the contents of it here, so we can try to see what's happening.
3. Then delete the file and restart the browser.

**Comment by shushtain** [comment]
`{"restartNow":false,"clientID":"02745009598464021","installTime":1618041600000,"usageData":{"created":1618126030560},"filtering":{"blockingLevel":2,"contentTypes":[],"exceptionDomains":[]},"darkMode":1,"siteTheme":false,"passwordManager":{"name":"none"},"filteringBlockedCount":1666,"keyMap":{"quitMin":["mod+q","mod+й"],"addTab":["mod+t","mod+е"],"addPrivateTab":["shift+mod+p","shift+mod+з"],"duplicateTab":["shift+mod+d","shift+mod+в"],"addTask":["mod+n","mod+т"],"toggleTasks":["shift+mod+e","shift+mod+у"],"goBack":"mod+left","enterEditMode":["mod+l","mod+k","mod+л","mod+д"],"runShortcut":["mod+e","mod+у"],"closeTab":["mod+w","mod+ц"],"restoreTab":["shift+mod+t","shift+mod+е"],"switchToNextTask":["mod+]","mod+ї","mod+ъ"],"toggleTabAudio":["shift+mod+m","shift+mod+ь"],"showHistory":["shift+mod+h","shift+mod+р"],"fillPassword":"mod+\\","showMenu":["mod+m","mod+ь"],"reload":["mod+r","mod+к"],"switchToPreviousTask":["mod+[","mod+х"]},"useSeparateTitlebar":false,"openTabsInForeground":false}`

**Comment by shushtain** [comment]
Okay, so now it's clear it starts to appear once I put shortcuts with Cyrillic letters in.

**Comment by shushtain** [comment]
I use both English and Ukrainian keysets, and it's really annoying when shortcuts don't work in some language. So please try to fix it. It worked perfectly in the previous updates.

**Comment by PalmerAL** [comment]
I think this is a regression from #1488, which happens when you have more than one shortcut set for `addTab`, `addPrivateTab`, etc. I think if you set only one shortcut it should work around the problem for now.

(cc @Milo123459 - root cause here is that if multiple shortcuts are set, the keyMap contains an array, which the implementation doesn't account for. I should be able to fix soon if you don't have time).

**Comment by shushtain** [comment]
Okay...

**Comment by shushtain** [comment]
I mean it works, but it's disappointing.

**Comment by Milo123459** [comment]
This is an error from the person actually building it from my understanding. You need to be on node 14>= just to be safe when building.

**Comment by PalmerAL** [comment]
@Milo123459 Nope, it's a separate issue from what we talked about before; it has nothing to do with the optional operator. What's going on is that it does this:

```
settings.get('keyMap').addTab.replace
```

But if you have more than one shortcut set, `addTab` here is actually an array, and calling `array.replace` doesn't work.

**Comment by Milo123459** [comment]
Oh that makes more sense.

**Comment by PalmerAL** [comment]
@shushtain I just published a new release (1.19.1); can you try installing that and confirm that it fixes the issue?

**Comment by shushtain** [comment]
> @shushtain I just published a new release (1.19.1); can you try installing that and confirm that it fixes the issue?

Can you point me to how to do that at least? If that's already public, I can't see any signs an update is available.

**Comment by PalmerAL** [comment]
Its on the github releases page: https://github.com/minbrowser/min/releases/tag/v1.19.1

**Comment by shushtain** [comment]
> Its on the github releases page: https://github.com/minbrowser/min/releases/tag/v1.19.1

Yep, it does fix multiple shortcuts issue, as well as sending usage statistics option, for me. Thank you!

**Comment by PalmerAL** [comment]
Great!
